src/interfaces/api/v4/credential.interface.ts

```diff
@@ -43,3 +46,36 @@ export interface ICredentialDisplayType {
   value: string | number;
   relationship_types?: string[];
 }
+
+export interface ICredentialFormatted {
+  create_timestamp: Date;
+  effective_date: Date;
+  inactive: boolean;
+  latest: boolean;
+  revoked: boolean;
+  revoked_date?: Date;
+  credential_id: string;
+  credential_type: ICredentialType;
+  addresses: ICredential[];
+  names: ICredentialName;
+  attributes: ICredentialAttribute[];
+  local_name: ICredentialName;
```

Comment:
I'm not sure if we want to carry over the concept of local name vs remote name from the previous implementation of OrgBook

Reply:
okay I removed local/remote name from credential and topic interfaces